openhands/linter/base.py

```diff
@@ -0,0 +1,75 @@
+from abc import ABC, abstractmethod
+
+from pydantic import BaseModel
+
+
+class LintResult(BaseModel):
+    file: str
+    line: int  # 1-indexed
+    column: int  # 1-indexed
+    message: str
+
+    def visualize(self, window_size: int = 10) -> str:
+        """Visualize the lint result by print out all the lines where the lint result is found."""
```

Comment:
I think it might be nice to have a test of just this function. I wanted to see an example of how it works, but I realized that there is no test of the function itself.

Also, could you please add arguments to the docstring? It's not clear from what "window_size" refers to (lines, characters, etc).

Reply:
done in most recent commit!